astroquery/vo_service_discovery/tests/shared_registry.py

```diff
@@ -0,0 +1,169 @@
+import os
+from astropy.table import Table
+from astroquery.vo_service_discovery import Registry
+
+
+class Helpers(object):
+    ""
+
+    DATA_FILES = {}
+
+    def table2ecsv(self, current, filepath):
+        """Dump a table and its meta data to ECSV"""
+        from astropy.io import ascii as ap_ascii
+        try:
+            ap_ascii.write(current, filepath, format='ecsv', overwrite=True)
+        except Exception as e:
```

Comment:
why the try/expect if the exception is raised anyway? There are a few other similar cases below, too.

Reply:
No idea why those were there.  Thanks for pointing them out.